Vagrantfile

```diff
@@ -109,71 +86,32 @@ $build_id = "#{$job_name}-#{$build_number}"
 # we can run VMs locally without having any the `build_id` in the name.
 if ENV['BUILD_NUMBER'] then
     $build_id_name = "-build-#{$build_id}"
-    $rsync_exclude = ".git"
-else
-    $rsync_exclude = "GIT_VERSION"
```

Comment:
If I understood this correctly, we'll be using guest additions also in the CI, so the .git folder is available during the CI run.

To make this change complete, we should remove the following line from `contrib/vagrant/start.sh`:
```
make GIT_VERSION envoy/SOURCE_VERSION
```

Reply:
Or better yet, force rm those files at the same place where the `make` command was. Having old versions of those files present during build can mess things up.